web-gateway/app/controllers/Main.java

```diff
@@ -38,36 +38,4 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
         );
     }
 
```

Comment:
Sorry... I had missed the changes to this file in your most recent commit. I thought only `UserController.java` had changed. Your most recent version is exactly what I had in mind, thanks!